app/src/screens/Splash.tsx

```diff
@@ -204,22 +205,42 @@ const Splash: React.FC = () => {
           })
 
           if (onboardingComplete(dataAsJSON) && !attemptData?.lockoutDate) {
-            navigation.navigate(Screens.EnterPIN as never)
+            navigation.dispatch(
+              CommonActions.reset({
```

Comment:
Cool. Didn't know we could do this.

Reply:
Yeah it's pretty handy! Took some researching to find how to do it